ui/component/or-attribute-history/src/index.ts

```diff
@@ -381,90 +396,136 @@ export class OrAttributeHistory extends translate(i18next)(LitElement) {
-                                grid: {
-                                    color: "#cccccc"
+                        {
+                            start: 0,
+                            end: 100,
+                            backgroundColor: bgColor,
+                            fillerColor: bgColor,
+                            dataBackground: {
+                                areaStyle: {
+                                    color: this._style.getPropertyValue("--internal-or-attribute-history-graph-fill-color")
                                 }
                             },
-                            x: {
-                                type: "time",
-                                min: this._startOfPeriod,
-                                max: this._endOfPeriod,
-                                time: {
-                                    tooltipFormat: 'MMM D, YYYY, HH:mm:ss',
-                                    displayFormats: {
-                                        millisecond: 'HH:mm:ss.SSS',
-                                        second: 'HH:mm:ss',
-                                        minute: "HH:mm",
-                                        hour: "HH:mm",
-                                        week: "w"
-                                    },
-                                    unit: this._timeUnits,
-                                    stepSize: this._stepSize
-                                },
-                                ticks: {
-                                    color: "#000",
-                                    font: {
-                                        family: "'Open Sans', Helvetica, Arial, Lucida, sans-serif",
-                                        size: 9,
-                                        style: "normal"
-                                    }
+                            selectedDataBackground: {
+                                areaStyle: {
+                                    color: this._style.getPropertyValue("--internal-or-attribute-history-graph-fill-color"),
+                                }
+                            },
+                            moveHandleStyle: {
+                                color: this._style.getPropertyValue("--internal-or-attribute-history-graph-fill-color")
+                            },
+                            emphasis: {
+                                moveHandleStyle: {
+                                    color: this._style.getPropertyValue("--internal-or-attribute-history-graph-fill-color")
                                 },
-                                gridLines: {
-                                    color: "#cccccc"
+                                handleLabel: {
+                                    show: true
                                 }
+                            },
+                            handleLabel: {
+                                show: false
                             }
```

Comment:
I tested this branch out locally to see what it looked like.
When I was dragging the slider left/right, I was quite annoyed by the tooltip showing time.
I don't think it's necessary, especially since it's cut off at the left- and right side of the chart.

There's an option in ECharts config for this;
```suggestion
                            },
                            showDetail: false
```

Reply:
good point, fixed